examples/without-map/app.js

```diff
@@ -0,0 +1,74 @@
+/* global window,document,fetch */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import {MapController} from 'deck.gl/dist/controllers';
```

Comment:
Why not expose `MapController` as a top-level export of `deck.gl` so you can just
`import {MapController} from 'deck.gl';`
and eliminate the change to webpack config?